sample_project/Assets/SampleViewer/Samples/OverviewMap/Scripts/ChangeOverviewScale.cs

```diff
@@ -0,0 +1,30 @@
+using Esri.HPFramework;
+using UnityEngine;
+
+public class ChangeOverviewScale : MonoBehaviour
+{
+    float CameraSize
+    {
+        get; 
+        set;
+    }
+
+    float LocationMarkerScale
+    {
+        get; 
+        set; 
```

Comment:
Having a setter here implies you could change it. This code doesn't have a way to change it at runtime.

Reply:
I thought we weren't making it so you can change it at run time? We would need to add UI Elements to change it during runtime. We shouldn't expect people to go into the inspector during runtime to change it.